exporters/stdout/stdoutmetric/exporter_test.go

```diff
@@ -84,7 +84,7 @@ func TestExporterHonorsContextErrors(t *testing.T) {
 		require.NoError(t, err)
 		return func(ctx context.Context) error {
 			var data metricdata.ResourceMetrics
-			return exp.Export(ctx, data)
+			return exp.Export(ctx, &data)
```

Comment:
I suggest changing the `data` variable initialization code (and add `&` or `new` there)

Reply:
updated as suggested